### DISCUSSION TITLE
Moderation

### DISCUSSION THREAD

**Discussion by scarpentier** [ClaperCo/Claper#92] [Ideas]
We really like Claper but we'd like a way to moderate the messages and questions in the Event Manager before it's shown to every attendee.

**Comment by alxlion** [comment]
No, that's not possible at the moment. We're relying mainly on the goodwill of the audience. If there are any problems, you can always exclude them permanently from your event. The only way to filter is with “pins”, but it is possible to display only pinned messages on the presentation screen.
